dwitter/templates/snippets/deleted_dweet_card.html

```diff
@@ -9,6 +9,6 @@
         {% include 'snippets/remix_info.html' %}
       {% endif %}
     </div>
-    <p class=dweet-timestamp> {{ dweet.posted }} </p>
+    <p class=dweet-timestamp>{{ dweet.posted | date:"Y-m-d H:i" }}</p>
```

Comment:
While working on this I ran into what I think is a bug with moment. The library doesn't seem to apply daylight savings time properly to dates parsed with an explicit format string:

```
moment.utc("Mar 12 2017 3:33 PM", "MMM D YYYY h:mm A").local().format()
> "2017-01-12T10:33:00-05:00"

moment.utc("2017-03-12 15:33").local().format()
> "2017-03-12T11:33:00-04:00"
```

That's why I switched the date format to this ISO 8601 date format, which is recognized automatically by moment.